libs/tup-components/src/layout/Sidebar/Sidebar.tsx

```diff
@@ -8,7 +8,9 @@ const Sidebar: React.FC = () => {
   return (
     <div className={styles['root']}>
       <Navbar>
-        <NavItem to={'/'}>Dashboard</NavItem>
+        <NavItem end icon="dashboard" to={'/'}>
+          Dashboard
+        </NavItem>
         <NavItem to={'/projects'}>Projects & Allocations</NavItem>
         <NavItem to={'/tickets'}>Tickets</NavItem>
         {loggedIn && <NavItem to={'/logout'}>Log Out</NavItem>}
```

Comment:
**Icons.**

These are not easy to find. I will add them to `/static/ui/index.html`. Hedda needs them too. So will TRAM.

```suggestion
        <NavItem icon="allocations" to={'/projects'}>Projects & Allocations</NavItem>
        <NavItem icon="multiple-coversation" to={'/tickets'}>Tickets</NavItem>
        {loggedIn && <NavItem icon="exit" to={'/logout'}>Log Out</NavItem>}
```